### ISSUE TITLE
[Docs] Is it worth adding some notes on general hosting advice?

### ISSUE THREAD

**Issue by mikeshardmind** [Cog-Creators/Red-DiscordBot#3926]
There is a lot that can go on with people hosting a bot. I don't think it would be reasonable to walk users through literally every aspect of administering their systems, but as a recent incident showed, even people who have been hosting large public bots for a while may run into some preventable issues for lack of knowing there was a way to prevent them.

In particular, information about setting resource usage limits (and linking out, not fully documenting this per OS) could help prevent certain things like out of memory errors from locking up the host as well, but there are other smaller things that may reasonably fit into this category as well.

**Comment by Jackenmen** [comment]
As long as we will only put an info with a link instead of full documentation, it shouldn't be much of a burden to us and I can see how it would be helpful to have it. Imo, this should also come with a note saying that it's not a complete info about every important aspect of system administration and it's only a guidance or something along these lines.
I don't know that much about administering systems myself, so I won't be of much help here.

I would mark as accepted but my contact on discord is currently limited so I'll leave that to someone else to avoid the chance of miscommunication ;)

**Comment by Dav-Git** [comment]
I agree that this would be a good idea. I'm not very experienced with it myself so something to just quickly read up on looks like a great addition to the docs.

**Comment by Stonedestroyer** [comment]
Stuff that could be mentioned is enabling firewall and enabling automatic security updates as an example. (Ubuntu). It does become a bit more tricky if we wanna make it more generic for all OSes. So not sure if we want guidance or actual guides how to simply do it. I also expect someone running FreeBSD / CentOS to know what they are doing so could show Ubuntu / Debian users how to do some security things.

**Comment by EternalllZM** [comment]
Does anyone think this would still be useful?
